Alignment/CTPPS/interface/StraightTrackAlignment.h

```diff
@@ -22,6 +22,8 @@
 #include "DataFormats/CTPPSReco/interface/CTPPSDiamondRecHit.h"
 #include "DataFormats/CTPPSReco/interface/CTPPSPixelRecHit.h"
 
+#include "DataFormats/CTPPSReco/interface/TotemRPUVPattern.h"
+#include "DataFormats/CTPPSReco/interface/TotemRPLocalTrack.h"
```

Comment:
Do you mean TotemRPLocalTrack.h? I was going to use that one at the beginning, but forgot to remove the include.